pkg/simple/client/devops/jenkins/pure_request.go

```diff
@@ -24,10 +30,41 @@ func (j *Jenkins) SendPureRequestWithHeaderResp(path string, httpParameters *dev
 
 	client := &http.Client{Timeout: 30 * time.Second}
 
+	header := httpParameters.Header
+	bearTokenArray := strings.Split(header.Get("Authorization"), " ")
+	bearFlag := bearTokenArray[0]
+	if err == nil && bearFlag == "Bearer" {
```

Comment:
it's unnecessary to have `err == nil` since we returned if not.

Reply:
Also better to use case insensitive compare
```suggestion
	if strings.ToLower(bearFlag) == "bearer" {
```